### DISCUSSION TITLE
'pimcore_admin_gdpr_dataobject_searchdataobjects' after upgrade pimcore

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#14055] [Q&A / Support]
Hi All,

We have upgraded the pimcore 6 to 10 and we face this error on the console.
![image](https://user-images.githubusercontent.com/60934941/212132715-0cdd406f-ee67-4218-b397-429f4c9a4886.png)

Anyone have any idea on this error?

**Comment by kingjia90** [comment]
Unfortunately no, didn't found the solution for this (see also https://github.com/pimcore/pimcore/issues/12165)  but the problem seemed caused by FOSJsRoutingBundle not getting loaded. 😞
